pro_wes/tasks/track_run_progress.py

```diff
@@ -97,7 +97,7 @@ def task__track_run_progress(
     response.pop("request", None)
     document: DbDocument = db_client.upsert_fields_in_root_object(
         root="run_log",
-        **response,
+        **dict(response),
```

Comment:
Perhaps adding a type hint where `response` is created would prevent the need to cast it to a dictionary? Same for the equivalent change below

Reply:
Since response is of type `RunLog`, I am typecasting it, if I declare it to be `Dict` problems arise down the code. `response: RunLog = wes_client.get_run(run_id=remote_run_id)`